Add -pipe

Because my ruby skills are still much too basic to do this efficiently, would somebody mind adding:
```shell
CFLAGS=-pipe
CXXFLAGS=-pipe
```
and optionally for the fortran compilers:
```shell
FCFLAGS=-pipe
FFLAGS=-pipe
```
This causes gcc to prefer memory instead of temporary files, seriously speeding up compilation times, while having no effect on the produced machine code. (On some computers with next to no memory, this can cause gcc to get killed by the kernel, but considering that basically all chromebooks have at least 2GB of memory and some swap, this won't be an issue.)

So here's the rub.  We can add CFLAGS like ```-pipe``` to CREW_OPTIONS, but that would override other CFLAGS given elsewhere in the package file, since ```+=``` doesn't work.

i.e., this is what happens with ```CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD} CFLAGS+=' -pipe' CXXFLAGS+=' -pipe' FCFLAGS+='-pipe' FFFLAGS+=' -pipe'"```
```
Building from source, this may take a while...
Rename all *.la files to *.la_tmp
configure: error: invalid variable name: `CFLAGS+'
zsh failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib64 --mandir=/usr/local/share/man --build=x86_64-cros-linux-gnu --host=x86_64-cros-linux-gnu --target=x86_64-cros-linux-gnu CFLAGS+=' -pipe' CXXFLAGS+=' -pipe' FCFLAGS+='-pipe' FFFLAGS+=' -pipe'` exited with 1
```

So probably best just to drop it into package scripts... (I'm already adding ```-flto``` as I go, so maybe this isn't too much of a problem.

However, we COULD add this to MESON options thus:
```
CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release -Dc_args='-fuse-ld=lld -pipe' -Dcpp_args='-fuse-ld=lld -pipe'"
CREW_MESON_LTO_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release   -Dcpp_args='-flto -fuse-ld=gold -pipe' -Dcpp_link_args='-flto' -Dc_arg
s='-flto -fuse-ld=gold -pipe' -Dc_link_args='-flto'"
```

--- a/lib/const.rb
+++ b/lib/const.rb
@@ -69,8 +69,7 @@ end
 
 CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD}"
 CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release -Dc_args='-fuse-ld=lld -pipe' -Dcpp_args='-fuse-ld=lld -pipe'"
-CREW_MESON_LTO_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release   -Dcpp_args='-flto -fuse-ld=gold -pipe' -Dcpp_link_args='-flto' -Dc_arg
-s='-flto -fuse-ld=gold -pipe' -Dc_link_args='-flto'"
+CREW_MESON_LTO_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release   -Dcpp_args='-flto -fuse-ld=gold -pipe' -Dcpp_link_args='-flto' -Dc_args='-flto -fuse-ld=gold -pipe' -Dc_link_args='-flto'"
 
 # Cmake sometimes wants to use LIB_SUFFIX to install libs in LIB64, so specify such for x86_64
 # This is often considered deprecated. See discussio at https://gitlab.kitware.com/cmake/cmake/-/issues/18640

--- a/lib/const.rb
+++ b/lib/const.rb
@@ -1,6 +1,6 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.7.1'
+CREW_VERSION = '1.7.2'
 
 ARCH_ACTUAL = `uname -m`.strip
 # This helps with virtualized builds on aarch64 machines
@@ -68,8 +68,9 @@ when 'x86_64'
 end
 
 CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD}"
-CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release -Dc_args='-fuse-ld=lld' -Dcpp_args='-fuse-ld=lld'"
-CREW_MESON_LTO_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release   -Dcpp_args='-flto -fuse-ld=gold' -Dcpp_link_args='-flto' -Dc_args='-flto -fuse-ld=gold' -Dc_link_args='-flto'"
+CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release -Dc_args='-fuse-ld=lld -pipe' -Dcpp_args='-fuse-ld=lld -pipe'"
+CREW_MESON_LTO_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuildtype=release   -Dcpp_args='-flto -fuse-ld=gold -pipe' -Dcpp_link_args='-flto' -Dc_arg
+s='-flto -fuse-ld=gold -pipe' -Dc_link_args='-flto'"
 
 # Cmake sometimes wants to use LIB_SUFFIX to install libs in LIB64, so specify such for x86_64
 # This is often considered deprecated. See discussio at https://gitlab.kitware.com/cmake/cmake/-/issues/18640

